### DISCUSSION TITLE
Print byte-offset  of matches instead of line numbers?

### DISCUSSION THREAD

**Discussion by Kareem-Emad** [BurntSushi/ripgrep#2375] [Q&A] [answered]
Hi, 
I was wondering if there is a way to show byte offsets instead of line numbers in search results :pray: 

For example in `grep`, I can achieve that by passing --byte-index flag 
```
 grep -F --byte-offset -e '-> some_thing'  ./file.txt
```

I'm looking up options and I can't find this mentioned in ripgrip docs, is this supported? are there any plans to support such option :pray:

**Comment by BurntSushi** [comment, marked as answer]
> I'm looking up options and I can't find this mentioned in ripgrip docs, is this supported?

Have you tried searching for `--byte-offset`? It literally has the same flag name (and short flag) as GNU grep.

```
       -b, --byte-offset
           Print the 0-based byte offset within the input file
           before each line of output. If -o (--only-matching)
           is specified, print the offset of the matching part
           itself.

           If ripgrep does transcoding, then the byte offset is
           in terms of the result of transcoding and not the
           original data. This applies similarly to another
           transformation on the source, such as decompression
           or a --pre filter. Note that when the PCRE2 regex
           engine is used, then UTF-8 transcoding is done by
           default
```
